From b818dec284a9b7310d260c7277c37af258be7a41 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 31 Dec 2019 15:11:50 +0530
Subject: [PATCH 0699/1852] fpga: zynqmp-fpga: fix compile warning in
 fpga_mgr_register

commit 6efa26f57b63ea9c3c3417b5d959e0cafa908c04 from
https://github.com/Xilinx/linux-xlnx.git

This fixes the compile warning as below:

drivers/fpga/fpga-mgr.c:911:2: warning: ISO C90 forbids mixed declarations and code [-Wdeclaration-after-statement]
911 | struct dentry *d, *parent;
		^~~~~~

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/fpga/fpga-mgr.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/fpga/fpga-mgr.c b/drivers/fpga/fpga-mgr.c
index e1ea80f97cff..bfeb8fd526ca 100644
--- a/drivers/fpga/fpga-mgr.c
+++ b/drivers/fpga/fpga-mgr.c
@@ -805,6 +805,9 @@ EXPORT_SYMBOL_GPL(devm_fpga_mgr_create);
 int fpga_mgr_register(struct fpga_manager *mgr)
 {
 	int ret;
+#ifdef CONFIG_FPGA_MGR_DEBUG_FS
+	struct dentry *d, *parent;
+#endif
 
 	/*
 	 * Initialize framework state by requesting low level driver read state
@@ -818,8 +821,6 @@ int fpga_mgr_register(struct fpga_manager *mgr)
 		goto error_device;
 
 #ifdef CONFIG_FPGA_MGR_DEBUG_FS
-	struct dentry *d, *parent;
-
 	mgr->dir = debugfs_create_dir("fpga", NULL);
 	if (!mgr->dir)
 		goto error_device;
-- 
2.31.1

